viewer/static/js/app/components/containers/experiment-metadata-container.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import dateFormat from 'dateformat';
+
+import ExperimentMetadata from '../views/experiment-metadata';
+
+class ExperimentMetadataContainer extends React.Component {
+  render() {
+    return <ExperimentMetadata {...this.props } />;
+  }
+}
+
+const mapStateToProps = function(store) {
+  const currentDataset = store.datasetState.currentDataset;
+
+  let numClients = 0;
+  let numPings = 0;
+  for (let populationName in currentDataset.populations) {
```

Comment:
if `populations` is an array we should iterate via `.map()` or an indexed for loop for performance